site/src/api/queries/workspaces.ts

```diff
@@ -266,6 +266,12 @@ export const startWorkspace = (
 export const cancelBuild = (workspace: Workspace, queryClient: QueryClient) => {
 	return {
 		mutationFn: () => {
+			// If workspace status is pending, include expect_status parameter
```

Comment:
The comment is just saying what the code is doing below. I would rather not having this comment at all, since I can see it in the code, or explaining the why we need to set the `expect_status` to pending.